pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -578,32 +596,8 @@ impl CustomScan for PdbScan {
                     .collect();
 
             let need_snippets = builder.custom_state().need_snippets();
```

Comment:
It seems like maybe this variable was already unused... and is still unused? Not sure what I'm missing here... but maybe we don't have warnings turned on for unused locals (only unused arguments).

Reply:
This is indeed used on line 122.